### PR TITLE
added new functionality full screen

### PR DESCRIPTION
**Pull request by gasbriones**
Hi @xiaolin, we are using your library in a south american media company, and we think it's great!
We would like to add the full screen functionality.
Please tell us what you think.

FYI: the full screen icon is not "standard", feel free to change it as you like.

Regards

### CODE REVIEW

File: example/app.js [hunk 1]
```diff
@@ -225,6 +227,14 @@ class App extends React.Component {
                 checked={this.state.slideOnThumbnailHover}/>
                 <label htmlFor='slide_on_thumbnail_hover'>slide on thumbnail hover (desktop)</label>
             </li>
+            <li>
+              <input
+                  id='fullscreen'
+                  type='checkbox'
+                  onChange={this._handleCheckboxChange.bind(this, 'fullscreen')}
+                  checked={this.state.fullscreen}/>
+              <label htmlFor='infinite'>fullscreen</label>
```
Comment (gasbriones) [inline example/app.js]: please change htmlFor='infinite' for htmlFor='fullscreen'

**Comment by xiaolin** [comment]
Thanks for the PR. I will take a closer look sometime this week.

**Comment by gaborluk** [comment]
Good one, @gasbriones, hope it makes it to the product. Any chance it would this week, @xiaolin? I'm thinking of using this in a project with a close deadline.

**Comment by xiaolin** [comment]
@gaborluk I should be able to find some time tonight or tomorrow night, I'll work on getting this and the other PRs released.

**Comment by gaborluk** [comment]
Thank you, @xiaolin, that would be tremendously appreciated. Please let me know in case there's a blocker and you need a helping hand with this, I'm happy to help out.

**Comment by alejodaraio** [comment]
@xiaolin Plz ! work it! i need it too to my proyect, its a nice feature!

**Comment by franciscok** [comment]
Woow! it'll be a nice feature! thanks guys!

**Comment by lechuzosanjuan** [comment]
Thank you for your contribution @gasbriones .

**Comment by xiaolin** [comment]
released v0.6.1

**Comment by gasbriones** [comment]
@xiaolin thank you for accepting my pull request , I am very happy to collaborate and use their excellent react component.
Greetings from Buenos Aires, Argentina

**Pull request closed without merging**